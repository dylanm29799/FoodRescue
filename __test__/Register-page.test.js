// __tests__/Register-page.test.js

import React from "react";
import Register from "../screens/RegisterScreen";
import renderer from "react-test-renderer";
import { firebaseConfig } from "../config";
import * as firebase from "firebase";

firebase.initializeApp(firebaseConfig);
test("renders correctly", () => {
  const tree = renderer.create(<Register />).toJSON();
  expect(tree).toMatchSnapshot();
});
