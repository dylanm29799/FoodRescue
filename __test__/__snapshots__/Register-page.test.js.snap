// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
    }
  }
>
  <View
    accessibilityIgnoresInvertColors={true}
    style={
      Object {
        "alignItems": "center",
        "height": "100%",
        "justifyContent": "center",
        "width": "100%",
      }
    }
  >
    <Image
      source={1}
      style={
        Array [
          Object {
            "bottom": 0,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          },
          Object {
            "height": "100%",
            "width": "100%",
          },
          undefined,
        ]
      }
    />
    <Text
      style={
        Object {
          "paddingTop": 428.57142857142856,
        }
      }
    />
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fff",
          "borderColor": "#c44536",
          "borderRadius": 15,
          "borderWidth": 3,
          "flexDirection": "row",
          "height": 45,
          "justifyContent": "center",
          "margin": 10,
          "paddingRight": 42.857142857142854,
        }
      }
    >
      <Text />
      <TextInput
        allowFontScaling={true}
        onChangeText={[Function]}
        placeholder="First Name"
        rejectResponderTermination={true}
        style={
          Object {
            "borderRadius": 20,
            "fontSize": 32.14285714285714,
            "marginRight": 21.428571428571427,
            "paddingHorizontal": 21.428571428571427,
            "paddingRight": 4.285714285714286,
            "paddingVertical": 10.714285714285714,
            "textAlign": "center",
            "width": "75%",
          }
        }
        underlineColorAndroid="transparent"
      />
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fff",
          "borderColor": "#c44536",
          "borderRadius": 15,
          "borderWidth": 3,
          "flexDirection": "row",
          "height": 45,
          "justifyContent": "center",
          "margin": 10,
          "paddingRight": 42.857142857142854,
        }
      }
    >
      <Text />
      <TextInput
        allowFontScaling={true}
        onChangeText={[Function]}
        placeholder="Last Name"
        rejectResponderTermination={true}
        style={
          Object {
            "borderRadius": 20,
            "fontSize": 32.14285714285714,
            "marginRight": 21.428571428571427,
            "paddingHorizontal": 21.428571428571427,
            "paddingRight": 4.285714285714286,
            "paddingVertical": 10.714285714285714,
            "textAlign": "center",
            "width": "75%",
          }
        }
        underlineColorAndroid="transparent"
      />
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fff",
          "borderColor": "#c44536",
          "borderRadius": 15,
          "borderWidth": 3,
          "flexDirection": "row",
          "height": 45,
          "justifyContent": "center",
          "margin": 10,
          "paddingRight": 42.857142857142854,
        }
      }
    >
      <Text />
      <TextInput
        allowFontScaling={true}
        autoCompleteType="email"
        keyboardType="email-address"
        onChangeText={[Function]}
        placeholder="Email"
        rejectResponderTermination={true}
        style={
          Object {
            "borderRadius": 20,
            "fontSize": 32.14285714285714,
            "marginRight": 21.428571428571427,
            "paddingHorizontal": 21.428571428571427,
            "paddingRight": 4.285714285714286,
            "paddingVertical": 10.714285714285714,
            "textAlign": "center",
            "width": "75%",
          }
        }
        underlineColorAndroid="transparent"
      />
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fff",
          "borderColor": "#c44536",
          "borderRadius": 15,
          "borderWidth": 3,
          "flexDirection": "row",
          "height": 45,
          "justifyContent": "center",
          "margin": 10,
          "paddingRight": 42.857142857142854,
        }
      }
    >
      <Text />
      <TextInput
        allowFontScaling={true}
        onChangeText={[Function]}
        placeholder="Password"
        rejectResponderTermination={true}
        secureTextEntry={true}
        style={
          Object {
            "borderRadius": 20,
            "fontSize": 32.14285714285714,
            "marginRight": 21.428571428571427,
            "paddingHorizontal": 21.428571428571427,
            "paddingRight": 4.285714285714286,
            "paddingVertical": 10.714285714285714,
            "textAlign": "center",
            "width": "75%",
          }
        }
        underlineColorAndroid="transparent"
      />
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fff",
          "borderColor": "#c44536",
          "borderRadius": 15,
          "borderWidth": 3,
          "flexDirection": "row",
          "height": 45,
          "justifyContent": "center",
          "margin": 10,
          "paddingRight": 42.857142857142854,
        }
      }
    >
      <Text />
      <TextInput
        allowFontScaling={true}
        onChangeText={[Function]}
        placeholder="Confirm Your Password"
        rejectResponderTermination={true}
        secureTextEntry={true}
        style={
          Object {
            "borderRadius": 20,
            "fontSize": 32.14285714285714,
            "marginRight": 21.428571428571427,
            "paddingHorizontal": 21.428571428571427,
            "paddingRight": 4.285714285714286,
            "paddingVertical": 10.714285714285714,
            "textAlign": "center",
            "width": "75%",
          }
        }
        underlineColorAndroid="transparent"
      />
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fff",
          "borderColor": "#c44536",
          "borderRadius": 15,
          "borderWidth": 3,
          "flexDirection": "row",
          "height": 45,
          "justifyContent": "center",
          "margin": 10,
          "paddingRight": 42.857142857142854,
        }
      }
    >
      <Text />
      <TextInput
        allowFontScaling={true}
        keyboardType="numeric"
        onChangeText={[Function]}
        placeholder="Your Personal Phone Number"
        rejectResponderTermination={true}
        style={
          Object {
            "borderRadius": 20,
            "fontSize": 32.14285714285714,
            "marginRight": 21.428571428571427,
            "paddingHorizontal": 21.428571428571427,
            "paddingRight": 4.285714285714286,
            "paddingVertical": 10.714285714285714,
            "textAlign": "center",
            "width": "75%",
          }
        }
        underlineColorAndroid="transparent"
      />
    </View>
    <Text>
      

    </Text>
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#c44536",
          "borderRadius": 10,
          "height": 50,
          "justifyContent": "center",
          "opacity": 1,
          "padding": 10,
        }
      }
    >
      <Text
        style={
          Object {
            "alignItems": "center",
            "color": "#EBF38B",
            "fontFamily": "MonM",
            "fontSize": 38.57142857142857,
            "textAlign": "center",
            "width": 321.42857142857144,
          }
        }
      >
        Register
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "marginBottom": -150,
          "paddingTop": 42.857142857142854,
        }
      }
    >
      <Text
        style={
          Object {
            "borderBottomColor": "#c44536",
            "borderBottomWidth": 1,
            "color": "#c44536",
            "fontFamily": "MonM",
            "fontSize": 21.428571428571427,
            "marginBottom": 42.857142857142854,
            "textAlign": "center",
            "width": 214.28571428571428,
          }
        }
      >
        Socials
      </Text>
      <View
        style={
          Object {
            "flexDirection": "row",
          }
        }
      >
        <Text />
        <Text />
        <Text />
      </View>
    </View>
  </View>
</View>
`;
